Update windows-agent-ddagent-user.md (#9228)
DataDog/documentation: content/en/agent/faq/windows-agent-ddagent-user.md

For example, if the directory check is monitoring a directory that has specific access rights, e.g. allowing only members of the Administrators group read access, then the existing Agent currently can monitor that directory successfully since `LOCAL_SYSTEM` has administrator rights. Upon upgrading, the administrator must add `ddagentuser` to the access control list for that directory in order for the directory check to function.

> Checking against the DHCP Server service on Windows Server OS in the Windows Service Integration doesn't work due to the special ACL for the `DHCPServer` service. The check will return `UNKNOWN` in such case.

Reviewer: ```suggestion
**Note**: For Windows Server OS, the Windows Service integration cannot check against the DHCP Server service due to the special ACL for the `DHCPServer` service. The check returns `UNKNOWN` in such case.
```
